bib/internal/uploader/aws.go

```diff
@@ -12,19 +11,11 @@ import (
 	"github.com/osbuild/images/pkg/cloud/awscloud"
 )
 
-// NewAWSClient returns an awscloud.AWS configured with a session for a given
-// region. It reads the credentials from environment variables:
-// AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY.
+// NewAWSClient returns an awscloud.NewDefault configured with a session for a given
+// region. It uses the default behavior of the aws library
+// like envionment variables, credential file... (see README.md)
 func NewAWSClient(region string) (*awscloud.AWS, error) {
-	keyID := os.Getenv("AWS_ACCESS_KEY_ID")
-	if keyID == "" {
-		return nil, fmt.Errorf("AWS_ACCESS_KEY_ID environment variable is required")
-	}
-	secretKey := os.Getenv("AWS_SECRET_ACCESS_KEY")
-	if secretKey == "" {
-		return nil, fmt.Errorf("AWS_SECRET_ACCESS_KEY environment variable is required")
-	}
-	return awscloud.New(region, keyID, secretKey, "")
+	return awscloud.NewDefault(region)
```

Comment:
I guess the function is useless now, we should just use `awscloud.NewDefault()` directly instead.